documentation/src/components/home.ts

```diff
@@ -12,15 +12,18 @@ class HomeElement extends LayoutElement {
         return html`
             <section id="hero">
                 <div class="spectrum-Article">
-                    <docs-spectrum-logo size="128px"></docs-spectrum-logo>
                     <h1 class="spectrum-Heading1--display">
                         Spectrum Web Components
```

Comment:
I think we should say "Meet Spectrum Web Components", like other members of the ring.